core/io/multiplayer_api.cpp

```diff
@@ -50,7 +50,7 @@ _FORCE_INLINE_ bool _should_call_local(MultiplayerAPI::RPCMode mode, bool is_mas
 			// Do nothing.
 		} break;
 		case MultiplayerAPI::RPC_MODE_REMOTE: {
-			// Do nothing also. Remote cannot produce a local call.
+			// Do nothing. Also, remote cannot produce a local call.
```

Comment:
The also refers to the previous "Do nothing." on line 50. This changes the meaning of the comment.
Simplest fix is to drop "also".